packages/react-cards/src/components/Card/Card.view.tsx

```diff
@@ -81,14 +81,24 @@ export const CardView: ICardComponent['view'] = props => {
     }
   );
 
+  const _onKeyDown = (ev: React.KeyboardEvent): void => {
+    if (onClick && (ev.which === KeyCodes.enter || ev.which === KeyCodes.space)) {
+      onClick(ev as any);
+      ev.preventDefault();
+    }
+  };
+
   return (
     <Slots.root
+      onClick={onClick}
+      onKeyDown={_onKeyDown}
```

Comment:
> onKeyDown={_onKeyDown} [](start = 5, length = 23)

If I understood this PR correctly, this should fix the issue for keyboard users. But what about screen reader users? How will they know that this is a clickable item? Is there a role being set? Or is there a clicktarget inside the card that will convey this information?

Reply:
@Anschur I fixed this issue, please review again.